app/views/content/early-years-teaching-training.md

```diff
@@ -91,25 +93,19 @@ A full-time bachelor’s degree in an early childhood-related subject leading to
 (SFE)](https://www.gov.uk/student-finance).
 
 $EY-graduate$
- 
+
 Bursaries are available, including:
 
  * £5,000 if you have a first class degree
  * £4,000 if you have a 2:1 degree
  * £2,000 if you have a 2:2 degree
- 
+
 $EY-graduate-employment$
 
-This one-year part-time route is for graduates working in an early years setting who need further training to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards). 
+This one-year part-time route is for graduates working in an early years setting who need further training to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
 
 Funding of up to £14,000 is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train.
 
-$EY-self-funded$ 
-
-This self-funded route is ideal for graduates with early-years experience. If you already meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards), you can gain EYTS without the need for further training. 
-
-## Next Steps
```

Comment:
What's the logic for removing a next steps section?

Reply:
there were just two links in this section - one link repeated a link further up the page, so I just deleted that one and move the other one up too

Reply:
Cool - I think this can be revisited as part of our CTA review for these 'offramp' pages.